content/docs/user-guide/project-structure/dvc-files.md

```diff
@@ -73,14 +73,16 @@ The following subfields may be present under `deps` entries:
-| `repo`                          | Only in external dependencies created with `dvc import`: It can contain `url`, `rev`, and `rev_lock` (detailed below).                                                                                                                           |
+| `repo`                          | Only in external dependencies created with `dvc import`: It can contain `url`, `rev`, `rev_lock`, `config` and `remote` (detailed below).                                                                                                        |
 
 ### Dependency `repo` subfields:
 
-| Field      | Description                                                                                                                                 |
-| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
-| `url`      | URL of Git repository with source DVC project                                                                                               |
-| `rev`      | Only when `dvc import --rev` is used: Specific commit hash, branch or tag name, etc. (a [Git revision]) used to import the dependency from. |
-| `rev_lock` | Git commit hash of the external <abbr>DVC repository</abbr> at the time of importing or updating the dependency (with `dvc update`)         |
+| Field      | Description                                                                                                                                                                                           |
+| ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `url`      | URL of Git repository with source DVC project                                                                                                                                                         |
+| `rev`      | Only when `dvc import --rev` is used: Specific commit hash, branch or tag name, etc. (a [Git revision]) used to import the dependency from.                                                           |
+| `rev_lock` | Git commit hash of the external <abbr>DVC repository</abbr> at the time of importing or updating the dependency (with `dvc update`)                                                                   |
+| `config`   | Only when `dvc import --config` is used: Path to a [config file](/doc/command-reference/config) that will be merged with the config in the target repository.                                         |
+| `remote`   | Only when `dvc import --remote` or `--remote-config` is used: name of the `dvc remote` to set as a default or remote config options to merge with a default remote's config in the target repository. |
```

Comment:
So both `--config` and `--remote` will persist but `--remote-config` will not? Can we explain it somewhere?

Reply:
yes, it will look like:
```
repo:
   ...
   remote:
      account_key=fasds
```

Reply:
I'll probably add more examples to `import` and will link that here.

Reply:
I see, it goes back to the discussion in https://github.com/iterative/dvc/pull/9747. So the idea is that if you have info you don't want stored in the repo, you need to use `--config` instead of `--remote`/`--remote-config`?

Reply:
@dberenbaum Yeah, correct.

Reply:
As discussed, let's explain for `import` how to use the options when you do and don't want those options committed to git.

Reply:
Added some examples.